Plugin crash when creating bigip_ltm_virtual_server on version 1.23.0

### Environment

 * TMOS/Bigip Version: BIG-IP 17.1.2 Build 0.0.8 Final
 * Terraform Version: v1.12.2
 * Terraform bigip provider Version: v1.23.0

### Summary

After updating to the version v1.23.0 of the f5 provider, trying to create a virtual server with the bigip_ltm_virtual_server resource causes a crash.

### Steps To Reproduce

The following example setup results in a plugin crash:

```
resource "infoblox_ip_allocation" "node" {
  fqdn      = "test123node.test.com"
  ipv4_cidr = "10.105.13.0/24"
  ttl       = 300
}

resource "infoblox_ip_allocation" "vs" {
  fqdn      = "test123vs.test.com"
  ipv4_cidr = "10.105.13.0/24"
  ttl       = 300
}

resource "bigip_ltm_node" "node" {
  name    = "/Common/test123node"
  address = infoblox_ip_allocation.node.fqdn
}

resource "bigip_ltm_pool" "pool" {
  name = "/Common/test123pool"
}

resource "bigip_ltm_pool_attachment" "attach" {
  pool = bigip_ltm_pool.pool.name
  node = "${bigip_ltm_node.node.name}:443"
}

resource "bigip_ltm_virtual_server" "vs" {
  name                       = "/Common/test123vs"
  destination                = infoblox_ip_allocation.vs.ipv4_addr
  pool                       = bigip_ltm_pool.pool.name
  port                       = 443
  ip_protocol                = "tcp"
  source_address_translation = "automap"
}
```

### Expected Behavior

As in previous versions (for example 1.22.10), this configuration should successfully create a virtual server.

### Actual Behavior

```
bigip_ltm_virtual_server.vs: Creating...
╷
│ Error: Plugin did not respond
│
│   with bigip_ltm_virtual_server.vs,
│   on test.tf line 27, in resource "bigip_ltm_virtual_server" "vs":
│   27: resource "bigip_ltm_virtual_server" "vs" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-bigip_v1.23.0 plugin:

panic: interface conversion: interface {} is nil, not int

goroutine 37 [running]:
github.com/F5Networks/terraform-provider-bigip/bigip.getVirtualServerConfig(0xc0005c9c80, 0xc0002eb608)
        github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_ltm_virtual_server.go:598 +0xf66
github.com/F5Networks/terraform-provider-bigip/bigip.resourceBigipLtmVirtualServerCreate({0x10d5840, 0xc0000c89a0}, 0xc0005c9c80, {0xf58400, 0xc000501880})
        github.com/F5Networks/terraform-provider-bigip/bigip/resource_bigip_ltm_virtual_server.go:294 +0x194
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc0003eee00, {0x10d5798, 0xc00056c5d0}, 0xc0005c9c80, {0xf58400, 0xc000501880})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/resource.go:707 +0x10f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc0003eee00, {0x10d5798, 0xc00056c5d0}, 0xc0005caf70, 0xc0005c9a80, {0xf58400, 0xc000501880})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/resource.go:837 +0xa8f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc00038e4f8, {0x10d5798?, 0xc00056c510?}, 0xc000479270)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.25.0/helper/schema/grpc_provider.go:1021 +0xd1a
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0xc000285360, {0x10d5798?, 0xc00038c8a0?}, 0xc0000c8460)
        github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/tf5server/server.go:818 +0x55f
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0xf13840, 0xc000285360}, {0x10d5798, 0xc00038c8a0}, 0xc0000c83f0, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.14.3/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:385 +0x1a6
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002785a0, {0x10da380, 0xc0000e8000}, 0xc00054d8c0, 0xc0004402a0, 0x177e500, 0x0)
        google.golang.org/grpc@v1.56.3/server.go:1335 +0xdb3
google.golang.org/grpc.(*Server).handleStream(0xc0002785a0, {0x10da380, 0xc0000e8000}, 0xc00054d8c0, 0x0)
        google.golang.org/grpc@v1.56.3/server.go:1712 +0x9cc
google.golang.org/grpc.(*Server).serveStreams.func1.1()
        google.golang.org/grpc@v1.56.3/server.go:947 +0xaf
created by google.golang.org/grpc.(*Server).serveStreams.func1 in goroutine 10
        google.golang.org/grpc@v1.56.3/server.go:958 +0x133

Error: The terraform-provider-bigip_v1.23.0 plugin crashed!
```

Leaving out the virtual server from the above configuration applies successfully.

--- a/bigip/resource_bigip_ltm_virtual_server.go
+++ b/bigip/resource_bigip_ltm_virtual_server.go
@@ -595,7 +595,7 @@ func getVirtualServerConfig(d *schema.ResourceData, config *bigip.VirtualServer)
 	config.Vlans = vlans
 	config.IPProtocol = d.Get("ip_protocol").(string)
 	config.TrafficMatchingCriteria = d.Get("trafficmatching_criteria").(string)
-	config.ConnectionLimit = d.Get("connection_Limit").(int)
+	config.ConnectionLimit = d.Get("connection_limit").(int)
 	srcAddrsTrans := struct {
 		Type string `json:"type,omitempty"`
 		Pool string `json:"pool,omitempty"`

